packages/platform-ios/src/lib/platformIOS.ts

```diff
@@ -61,7 +71,16 @@ export const platformIOS =
     return {
       name: '@rnef/platform-ios',
       description: 'RNEF plugin for everything iOS.',
-      autolinkingConfig: { project: { ...iosConfig } },
+      autolinkingConfig: {
+        get project() {
```

Comment:
Hmmm, could you check if this calculate prop (and also the android version of it), is read by some downstream code, like Joi schema checking or perhaps verbose output, etc?

Reply:
we're not making schema validation for plugins yet, but we should introduce it at some point for platforms definitely